e2e/config/docker/docker-compose.test.services.yml

```diff
@@ -29,7 +29,7 @@ services:
       service: base-service
     working_dir: /opt/orbs/projects/services/public-api-service-typescript/
     environment:
-      - validateSubscription=true
+      VALIDATE_SUBSCRIPTION: 'true'
```

Comment:
AFAIK, the quotes are not necessariy

Reply:
I'm pretty sure without quotes it will fail on `docker-compose up`

Reply:
Really? `true` is a valid value in a YAML. Oh well.